orca-igor/orca-igor.gradle

```diff
@@ -22,6 +22,7 @@ dependencies {
   compile project(":orca-front50")
   compile spinnaker.dependency('kork')
   compile spinnaker.dependency('bootAutoConfigure')
+  compile 'com.google.apis:google-api-services-cloudbuild:v1-rev836-1.25.0'
```

Comment:
Removed when we bump spinnaker-dependencies?

Reply:
Yup, exactly.  I'll remove it in this same PR after I rebase on top of the bump.